src/NUnitFramework/framework/Internal/CultureDetector.cs

```diff
@@ -76,7 +76,7 @@ public bool IsCultureSupported( CultureAttribute cultureAttribute )
             //catch( ArgumentException ex )
             //{
             //    reason = string.Format( "Invalid culture: {0}", ex.ParamName );
-            //    return false; 
+            //    return false;
```

Comment:
I think we should rather remove the commented code

Reply:
good point. done